creator-node/src/services/sync/syncUtil.ts

```diff
@@ -0,0 +1,126 @@
+import type Logger from 'bunyan'
+
+import axios from 'axios'
+import _ from 'lodash'
+
+const asyncRetry = require('../../utils/asyncRetry')
+
+const EXPORT_REQ_TIMEOUT_MS = 10000 // 10000ms = 10s
```

Comment:
unrelated i think this timeout is causing some failures - maybe we could just increase to ~30sec?

Reply:
sounds good to me! in one implementation it was 10s, in the other it was 5 minutes! also maybe we should lower the time issueSyncReq monitors since I'm assuming that's where issueSyncReq's 5 minutes came from?

Reply:
lmao

Reply:
that accounts for queue delays, export, then subsequent sync (db + disk writes) etc
lets handle that separately - i'd really love to dig into some metrics first

also if previously one was at 5min maybe this should be smth higher either just 5min or 1min

Reply:
ok I'll make it 1min for now and see if that causes any issues. btw here's the code pointer for the previous 5min: https://github.com/AudiusProject/audius-protocol/pull/3771/files#diff-1c3f91243ef085b0e59d4225bcbff6c995a63649898ff1d269efecea4006eaceL173

Reply:
coo